DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -94,7 +94,7 @@ public void GetAllDelegateUsersByUsername_searches_AliasID()
         public void GetDelegateUsersByEmailAddress_Returns_delegate_user()
         {
             // Given
-            var expectedDelegateUser = UserTestHelper.GetDefaultDelegateUser();
+            var expectedDelegateUser = UserTestHelper.GetDefaultDelegateUser(resetPasswordId: 1);

```

Comment:
Couple of things with this.

Firstly, do you think we should have a similar test for admin users to make sure their reset password Id is properly retrieved?

Secondly, I feel like the default values for the GetDefaultDelegateUser should match the values of the record with the default Id. In this case the resetPasswordId should be 1 by default. This would then prevent any data tests that query the same user from mysteriously failing. But then on the other hand, that _might_ cause other tests to fail that create users based on the default delegate. But I'm not too sure on that. Thoughts?